server/model/picture.js

```diff
@@ -12,6 +13,7 @@ export type Picture = {|
   copyrightHolder?: string;
   fileFormat?: string;
   url?: ?string;
+  attribute?: {title: ?string, author: ?string, source: ?string, license: LicenseType};
```

Comment:
As mentioned - this might be nicer on the root of the object.

Reply:
I was inclined to leave it as is – it made sense to me to be grouped together.

Reply:
I don't think it does - attribution isn't really a type - but rather a way of displaying properties of an image.

Reply:
Fair – I'll update.